lib/client.js

```diff
@@ -557,6 +557,36 @@ class Client {
     });
   }
 
+
+  /**
+   * List Translation Requests. Callback is called with an map of
+   * TR access objects.
+   *
+   * @param {Object} [opts={}] - optional map of options
+   * @param {TranslationRequest~getTranslationRequestsCallback} cb - callback yielding a map of Translation Requests
+   */
+  docTrs(opts, cb) {
+    opts = utils.fixArgs(arguments);
+    if ((cb = arguments[1])) {
+      return utils.depromise(this.docTrs(opts), cb);
+    }
+    var serviceInstance = this.getServiceInstance(opts);
```

Comment:
no linter? Using `var`?

Reply:
linter, but lots of `var` still. Fixed this instance.